### ISSUE TITLE
Trying to load non-existing default icon

### ISSUE THREAD

**Issue by juho-jaakkola** [sbwtech/sbw_campaigns#7]
River item's media section is trying to load a non-existing image `/cache/0/default/icons/default/master.png`. 
```
Failed to load resource: the server responded with a status of 403 (Forbidden)
```

![image](https://user-images.githubusercontent.com/883920/31863731-24bb1fd6-b752-11e7-9731-fb6506b5be93.png)

It's probably is supposed to point to one of the files in `/_graphics/icons/default/default<size>.png`, but the `master` size is in fact missing from there. Then again I don't think campaigns should be using any default image (especially none of the ones in that directory).

**Comment by michelesbw** [comment]
I agree, there should be no default icon, at least an image or video should be added by user...campaigns with none of them are simply not possible to promote online
